.github/workflows/manual-sync.yml

```diff
@@ -1,6 +1,6 @@
 name: Sync with ppy:master
 
-on:
+"on":
```

Comment:
I've never seen this style used in a yaml file for github workflows or otherwise. What's the deal.

Reply:
it's a weird gotcha that yaml keys can be parsed as other types and so this can be the boolean "true", which is why yamllint recommends quoting them like this. but checking again, I misunderstood which yaml parsers are used in our projects and it looks like this doesn't cause any problems so I'll change it back